.gitlab-ci.yml

```diff
@@ -62,6 +62,25 @@ caramel:flake:
   script:
     - flake8 caramel/ tests/
 
+caramel:mypy:
+  stage: test
+  when: always
+  image: ${PYTHON_IMAGE}
+  before_script:
+    - pip3 install mypy sqlalchemy2-stubs
```

Comment:
This one still seems wrong.  sqlalchemy2-stubs is from  dropbox and is not the same stubs that the official sqlalchemy has, including the plugin. It's also mostly unmaintained ( I know from experience ).

It should be `SQLAlchemy[mypy] >= 1.4`   or `~= 1.4` instead.

Reply:
Reason for this is that pip install sqlalchemy[mypy] just gave me errors saying it couldn't find it, turns out you have to put it in quotes( "sqlachemy[mypy]"). Fixed